src/viur_cli/conf.py

```diff
@@ -169,6 +179,16 @@ def migrate(self):
 
         # conf updates must increase format version
         self.save()
-
+def print_changelog_from_github(user, repo):
```

Comment:
add two blank lines (PEP8!)
```suggestion


def print_changelog_from_github(user, repo):
```

Reply:
Generally, this function looks to be useful generally to obtain the changelog, also of other projects. Why not put it into the `utils` module?